force-app/main/default/lwc/apexError/apexError.html

```diff
@@ -0,0 +1,13 @@
+<template>
+    <lightning-card title="ApexError" icon-name="custom:custom63">
+        <template if:true={contact.data}>
```

Comment:
@pozil what do you think about this for a component that always displays an error message. I felt I had two options: 
1. Just display the error message, no conditional rendering in the template. 
2. Have conditional rendering, even though we will never show the non-error state.

I opted for number 2, so that there could be no confusion as to the typical structure of a component being that you only display the error condition when there is an error. 

I'd like your thoughts on this, both on the choice to go for option 2, and whether I need the never-displayed "this should not appear" block. Thanks!

Reply:
I'm good with option 2.

Should we also create another recipe that handles imperate apex errors coming from the same apex controller method? I think it's interesting to demonstrate a rejected promise.

Reply:
The imperative error is a good idea. I'll do that. 

I'll also add some comments in the template to communicate why it is the way it is.